### PR TITLE
fix: look for lowercase input schema in default paths

### PR DESCRIPTION
**Pull request by mvolfik**
recreated #646

### CODE REVIEW

File: src/lib/input_schema.ts [hunk 1]
```diff
@@ -49,18 +54,14 @@ export const readInputSchema = async (
...
-			inputSchemaPath: join(cwd, DEFAULT_INPUT_SCHEMA_PATHS[0]),
-		};
-	}
-
-	if (existsSync(join(cwd, DEFAULT_INPUT_SCHEMA_PATHS[1]))) {
-		return {
-			inputSchema: getJsonFileContent(join(cwd, DEFAULT_INPUT_SCHEMA_PATHS[1])),
-			inputSchemaPath: join(cwd, DEFAULT_INPUT_SCHEMA_PATHS[1]),
-		};
+	for (let i = 0; i < DEFAULT_INPUT_SCHEMA_PATHS.length; i++) {
```
Comment (vladfrangu) [inline src/lib/input_schema.ts:57]: Could use a simple for-of loop

**Review by vladfrangu — APPROVED** [review]

**Comment by vladfrangu** [comment]
@B4nan we should probably target the 0.20 branch and forward port to master, right?

**Comment by B4nan** [comment]
Well, either way, we want it in master, and then we can cherry-pick to 0.20 branch.

File: src/lib/input_schema.ts (hunk 1 above)
Comment (mvolfik) [reply src/lib/input_schema.ts]: my brain wasn't braining

**Pull request merged**